lib/messager.rb

```diff
@@ -0,0 +1,28 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Messager
+  # put your own credentials here
+  def initialize
+    account_sid = ENV['account_sid']
+    auth_token = ENV['auth_token']
+    @client = Twilio::REST::Client.new account_sid, auth_token
+  end
+# set up a client to talk to the Twilio REST AP
+  def send_sms(msg)
+    text_customer(msg)
+  end
+
+  private
+    def text_customer(x)
+      t = Time.now + 3600
+
+      @client.account.messages.create({
+      :from => ENV['from'],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)